cli/parsefile.go

```diff
@@ -200,3 +207,36 @@ func extractPmt(buf io.Reader, pid uint16) (psi.PMT, error) {
 	}
 	return pmt, nil
 }
+func verifySync(buf io.Reader) (bool,int64){
+   //function find the first sync byte of the array   
+  	pkt := make([]byte, 1)
```

Comment:
`pkt` feels like a poor name for this variable since it's only a single byte.  Throughout the rest of the code `pkt` refers to a full ts packet.  Something other than packet, such as `data` or even as simple as `b` seems to fit the context better